src/content_api/content_api_extract.py

```diff
@@ -100,7 +36,7 @@ def save_all_to_file(json_dict, page_links, related_page_links, collection_links
     df_rel['num_emb'] = df_rel['embedded_links'].map(len)
```

Comment:
not sure if you need `df_rel = df_rel[['url', 'embedded_links', 'related_links', 'collection_links']]`, correct me if I'm wrong